test/jdk/java/text/Format/NumberFormat/PositionTest.java

```diff
@@ -39,15 +33,71 @@
   Taligent is a registered trademark of Taligent, Inc.
 */
 
-import java.text.*;
-import java.io.*;
+/*
+ * @test
+ * @bug 4109023 4153060 4153061 8366400
+ * @summary test ParsePosition and FieldPosition
+ * @run junit PositionTest
+ */
 
 import org.junit.jupiter.api.Test;
 
+import java.text.DecimalFormat;
+import java.text.FieldPosition;
+import java.text.NumberFormat;
+import java.text.ParsePosition;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
 import static org.junit.jupiter.api.Assertions.fail;
 
 public class PositionTest {
 
+    // Parsing text which contains un-parseable data, but the index
+    // begins at the valid portion. Ensure PP is properly updated.
+    @Test
+    public void modifiedPositionTest() {
```

Comment:
JUnit5 tests default to package-private visibility, but I think aligning with other tests is more important.